services/src/datasets/external/gbif.rs

```diff
@@ -197,6 +300,217 @@ impl GbifDataProvider {
             .collect::<Vec<_>>())
     }
 
+    async fn get_datasets_autocomplete_items(
```

Comment:
why is this function never used?

Reply:
Removed in https://github.com/geo-engine/geoengine/commit/e23802633dc6b4ae701c3913186079f906872523 since https://github.com/geo-engine/geoengine/commit/37f6862cc45f1941c5189369051b38a83c79c72b made it obsolete